docs/crds/hostrule.md

```diff
@@ -309,6 +309,105 @@ The `networkSecurityPolicy` setting, in addition to any other parameters provide
+4. To overcome the above limitation for secure SNI and secure SNI dedicated virtual services, the user should always define a hostrule with **useRegex** set to **True** if regular expression paths are defined in secure Ingress/Route. If **useRegex** is **True**, AKO will create the pool with an encoded name.
+
+#### Express Application Root Redirect path
+If the application root path, i.e., path `/`, for a given host, defined in Ingress or Route, is exposed on a different path in the backend and needs to be redirected, then set this field to the required path in a hostrule to redirect requests. This is an optional field. The host and query strings in the redirect URI will be kept the same. The port in the redirect request will be set as the front-end port specified in the ingress, and the redirect protocol will be set based on the redirect port (HTTPS for ports 443 and 6443 and HTTP for any other port).
+
+        fqdn: test-avi.vmware.com
+        fqdnType: Exact
+        applicationRootPath: "/foo"
+
+If any other path (along with the application root **/**) is also specified for the host in the ingress/route, then requests for that path will be routed normally (without any redirect) by Avi. The path that is used as the **applicationRootPath** should not also be used as a regular path in the ingress/route, as it will lead to ambiguity while backend selection in Avi.
+
+        apiVersion: networking.k8s.io/v1
+        kind: Ingress
+        metadata:
+          name: test-ingress
+        spec:
+          rules:
+          - host: test-avi.vmware.com
+            http:
+              paths:
+              - backend:
+                  service:
+                    name: my-app-0
+                    port:
+                      number: 80
+                path: /bar
+                pathType: ImplementationSpecific
+              - backend:
+                  service:
+                    name: my-app-1
+                    port:
+                      number: 443
+                path: /
+                pathType: Exact
+          tls:
+          - hosts:
+            - test-avi.vmware.com
+            secretName: ingress-secret
+
+For the above Ingress definition, if we create a hostrule with `applicationRootPath` set to `/foo`, then :
+1. A request that comes on the URL **https://test-avi.vmware.com/** will first be redirected to **https://test-avi.vmware.com:443/foo** and then routed to the backend **my-app-1**.
+2. A request that comes on the URL **https://test-avi.vmware.com/bar** will be routed to backend **my-app-0**.
+
+***Note***
+1. This property is available only in HostRule `v1beta1` schema definition.
+2. The `applicationRootPath` field will not be supported or insecure ingress/route when AKO is running in SNI mode or SNI Dedicated mode.
```

Comment:
will not be supported or insecure ingress/route -> will not be supported for insecure Ingress/Route.

Reply:
Modified.

Reply:
replaced or with for.